src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,16 @@
+package shop;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class RealItemTest {
+    private static final double WEIGHT = 1560;
+
+    @Test
+    void setWeight() {
+        RealItem car = new RealItem();
```

Comment:
RealItem class is designed as simple model without any business logic. It doesn't make sense to write tests for getters/setters. The only method, which can be covered with test, is toString in this case. This task has 2 correct resolutions:
- don't write tests
- write test for toString